src/AIS.hpp

```diff
@@ -10,10 +10,15 @@
 #include <marnav/ais/message_01.hpp>
 #include <marnav/ais/message_05.hpp>
 
+#include <base/samples/RigidBodyState.hpp>
+#include <gps_base/BaseTypes.hpp>
+#include <gps_base/UTMConverter.hpp>
+#include <optional>
```

Comment:
Be careful with what you include in .hpp files, you could create a false dependency. I think the only thing you need in this .hpp is the utm converter. The rest may be needed by the .cpp and should be included just there 
```suggestion
#include <gps_base/UTMConverter.hpp>
```